apis/eventstream/index.yaml

```diff
@@ -91,6 +92,32 @@ get:
               value: |
                 event: light_client_optimistic_update
                 data: {"version":"phase0", "data": {"attested_header": {"beacon": {"slot":"1", "proposer_index":"1", "parent_root":"0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2", "state_root":"0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2", "body_root":"0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2"}}, "sync_aggregate": {"sync_committee_bits":"0x01", "sync_committee_signature":"0x1b66ac1fb663c9bc59509846d6ec05345bd908eda73e670af888da41af171505cc411d61252fb6cb3fa0017b679f8bb2305b26a285fa2737f175668d0dff91cc1b66ac1fb663c9bc59509846d6ec05345bd908eda73e670af888da41af171505"}, "signature_slot":"1"}}
+            payload_attributes:
+              description: |
+                The node has computed new payload attributes for execution payload building.
+
+                This event gives block builders and relays sufficient information to construct or
+                verify a block at `proposal_slot`. The meanings of the fields are:
+
+                - `version`: the identifier of the beacon hard fork at `proposal_slot`, e.g.
+                  `"bellatrix"`, `"capella"`.
+                - `proposal_slot`: the slot at which a block using these payload attributes may be
+                   built.
+                - `parent_block_root`: the beacon block root of the parent block to be built upon.
+                - `parent_block_number`: the execution block number of the parent block.
+                - `parent_block_hash`: the execution block hash of the parent block.
+                - `proposer_index`: the validator index of the proposer at `proposal_slot` on
+                   the chain identified by `parent_block_root`.
+                - `payload_attributes`: beacon-style encoding of `PayloadAttributesV<N>` as
```

Comment:
I doubt the term `beacon-style encoding` will be understandable. I would rephrase this as `Beacon API encoding`.

Reply:
thanks for the suggestion, I've tried to improve the wording in this commit 64815e233e5ebbceb4465c9d0d39f02461a0c5d6